BoxBrowseSDK/BoxBrowseSDK/UIImage+BOXBrowseSDKAdditions.m

```diff
@@ -10,20 +10,37 @@
 
 @implementation UIImage (BOXBrowseSDKAdditions)
 
-+ (UIImage *)imageFromBrowseSDKResourceBundleNamed:(NSString *)string
++ (UIImage *)box_imageFromBrowseSDKResourceBundleNamed:(NSString *)name
 {
-    NSString *str = [[[self resourcesBundle] resourcePath] stringByAppendingPathComponent:[[self resourcesBundle] pathForResource:string ofType:nil]];
-    return [UIImage imageWithContentsOfFile:[str stringByAppendingPathComponent:[NSString stringWithFormat:@"%@.png", string]]];
+    UIImage *imageFromBundle = nil;
+    NSBundle *bundle = [self resourcesBundle];
+
+    if ([self respondsToSelector:@selector(imageNamed:inBundle:compatibleWithTraitCollection:)]) {
+        imageFromBundle = [self imageNamed:name inBundle:bundle compatibleWithTraitCollection:nil];
+    } else {
+        NSString *path = [[bundle resourcePath] stringByAppendingPathComponent:[bundle pathForResource:name ofType:nil]];
+
+        NSString *fileName = nil;
+        if ([UIScreen mainScreen].scale == 2.0) {
```

Comment:
No need to handle @3x, all @3x devices only run iOS 8 and newer and thus support imageNamed:inBundle:compatibleWithTraitCollection: